src/controller.coffee

```diff
@@ -3948,9 +3961,8 @@ hook 'populate', 2, ->
     @session.viewports.palette.x = @paletteScroller.scrollLeft
 
 Editor::resizeMainScroller = ->
-  @mainScroller.style.width = "#{@dropletElement.clientWidth}px"
-  @mainScroller.style.height = "#{@dropletElement.clientHeight}px"
```

Comment:
was this line never needed? or is just not needed now that you are specifying `bottom` and `top` in other places?

Reply:
top, bottom, left, and right were already set in droplet.css. This line may have been needed at some point, but isn't anymore.